manifests/plugin/remote_execution/ssh.pp

```diff
@@ -49,6 +52,23 @@
     template_path => 'foreman_proxy/plugin/remote_execution_ssh.yml.erb',
   }
 
+  if $ssh_kerberos_auth {
+    if $::osfamily == 'RedHat' {
+      $rh_ruby_prefix = 'rubygem'
```

Comment:
I wonder if this additional variable is actually needed. I'd just inline string below.